.github/workflows/lintcommits.yml

```diff
@@ -62,7 +65,10 @@ jobs:
             const commits = await github.paginate(opts);
 
             const errors = [];
-            for (const { sha, commit: { message } } of commits) {
+            for (const { sha, commit: { message }, author } of commits) {
+              if (author.login in excluded_bot_ids && excluded_bot_ids[author.login] === author.id) {
```

Comment:
Not sure why we would need to check both the login and the ID?

Reply:
I figured if either changed we might want to check out what was going on - I made a simpler check based on an array, lmk if that's better.